tests/effective_dps_test.cpp

```diff
@@ -286,41 +286,41 @@ static void check_staves( const std::function<Approx( const std::string & )> &ca
 static void check_spears( const std::function<Approx( const std::string & )> &calc_expected_dps )
 {
     SECTION( "spears" ) { // typical value around 24
-        CHECK( calc_expected_dps( "spear_steel" ) == 24.5 );
+        CHECK( calc_expected_dps( "spear_steel" ) == 23.19 );
         CHECK( calc_expected_dps( "pike" ) == 23.0 );
-        CHECK( calc_expected_dps( "qiang" ) == 23.0 );
-        CHECK( calc_expected_dps( "spear_dory" ) == 23 );
-        CHECK( calc_expected_dps( "spear_homemade_halfpike" ) == 20.0 );
+        CHECK( calc_expected_dps( "qiang" ) == 26.54 );
+        CHECK( calc_expected_dps( "spear_dory" ) == 19.97 );
+        CHECK( calc_expected_dps( "spear_homemade_halfpike" ) == 18.4 );
         CHECK( calc_expected_dps( "spear_copper" ) == 19.0 );
-        CHECK( calc_expected_dps( "spear_pipe" ) == 19.0 );
-        CHECK( calc_expected_dps( "spear_knife_superior" ) == 18.0 );
-        CHECK( calc_expected_dps( "spear_knife" ) == 18.0 );
+        CHECK( calc_expected_dps( "spear_pipe" ) == 17.37 );
+        CHECK( calc_expected_dps( "spear_knife_superior" ) == 14.65 );
+        CHECK( calc_expected_dps( "spear_knife" ) == 14.65 );
         CHECK( calc_expected_dps( "pike_inferior" ) == 16.0 );
-        CHECK( calc_expected_dps( "spear_wood" ) == 15.0 );
-        CHECK( calc_expected_dps( "pitchfork" ) == 15.0 );
-        CHECK( calc_expected_dps( "spear_stone" ) == 14.0 );
-        CHECK( calc_expected_dps( "spear_forked" ) == 14.0 );
-        CHECK( calc_expected_dps( "pike_fake" ) == 8.5 );
+        CHECK( calc_expected_dps( "spear_wood" ) == 10.95 );
+        CHECK( calc_expected_dps( "pitchfork" ) == 12.48 );
+        CHECK( calc_expected_dps( "spear_stone" ) == 10.42 );
+        CHECK( calc_expected_dps( "spear_forked" ) == 12.15 );
+        CHECK( calc_expected_dps( "pike_fake" ) == 7.55 );
     }
 }
 
 static void check_polearms( const std::function<Approx( const std::string & )> &calc_expected_dps )
 {
     SECTION( "polearms" ) { // typical value around 35
```

Comment:
I'd stay at most a point under the typical value, I recommend dropping all the ones that popped up into the 40s down into the 35-36 range.

Reply:
Okay, nerfed them and we'll see if they get lowered enough.